Wikipedia/UI-V5/WMFArticleContainerViewController+TOC.swift

```diff
@@ -3,6 +3,20 @@ import Foundation
 import BlocksKit
 
 extension WMFArticleContainerViewController : WMFTableOfContentsViewControllerDelegate {
+    
+    public func tableOfContentsControllerWillDisplay(controller: WMFTableOfContentsViewController){
```

Comment:
:+1: but wondering if we should instead ask the delegate what the current item is, and let the ToC implement "selection on appear" instead of asking its delegate to do it. this way, delegates don't need to repeat implementations of "selection on appear."  nbd now since there's only 1 delegate